keyboards/cornia/keymaps/fire/tap_dances.c

```diff
@@ -0,0 +1,88 @@
+/* Copyright 2024 Vaarai
```

Comment:
This keymap would not compile on latest `qmk_firmware`, as tap dance is now hooked up to introspection the default config assumes they are still within `keymap.c`.

Reply:
I used `extern tap_dance_action_t tap_dance_actions[TAP_DANCE_ACTIONS_COUNT];` in `tap_dances.h` and include it in `keymap.c`. 

Do you want me to move `tap_dance_actions` declaration & definition to `keymap.c` ?